app/mailers/user_mailer.rb

```diff
@@ -186,7 +186,7 @@ def invite_request_declined(user_id, total, reason)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: @user.email,
-        subject: t('user_mailer.invite_request_declined.subject', app_name: ArchiveConfig.APP_SHORT_NAME)
+        subject: t("user_mailer.invite_request_declined.subject", app_name: ArchiveConfig.APP_SHORT_NAME)
```

Comment:
Should this also be using `default_i18n_subject`?

Reply:
Note to self, probably yes, still needs fixing and testing.